pydrive2/auth.py

```diff
@@ -296,14 +296,21 @@ def ServiceAuth(self):
         if set(self.SERVICE_CONFIGS_LIST) - set(self.client_config):
             self.LoadServiceConfigSettings()
         scopes = scopes_to_string(self.settings["oauth_scope"])
+        client_service_json = self.client_config.get("client_service_json")
```

Comment:
qq - should we name it `client_service_json_file_path` (similar to the previous convention?)

Reply:
makes sense!